constitution/README.md

```diff
@@ -1 +1,44 @@
-# Fix Planet Club - Constitution
+---
+title:  "Charitable Incorporated Organisation Constitution for _Fix Planet Club_"
+---
+
+
+1.) **Name**
+
+> The name of the Charitable Incorporated Organisation ("the CIO") is:
+>
+> Fix Planet Club
+
+
+2.) **National location of principal office**
+
+> The CIO principle office is in Scotland.
+
+3.) **Objects**
+
+> The objects of the CIO are:
+>
+> 1. For the public benefit, build and support a community around sustainable
+>    climate technology projects, both individual and commercial; and
+>
+> 2. For the public benefit, fund and distribute a no-strings attached grant
+>    to individuals working on a project related to climate technology that
+>    addresses at least one of the following criteria:
+>
+>     a. Is innovative in the industry, or
+>     b. Provides value to the local community, or
+>     c. Addresses marginalised communities, or
+>     d. Provides value to the individual themselves, or
```

Comment:
Are we sure we would like to fund projects valuable only for a single individual?

Reply:
@marcosh Yeah, i wasn't quite sure how to word this part.

What I had in mind was, suppose a project comes in that doesn't really add anything new, _but_ is a growth opportunity for the specific person. I kind had in mind some of the ideas in this blog post - https://jvns.ca/blog/2017/02/16/hiring---opportunity/ 

i.e. is the opportunity _good for the person_?

But maybe, upon reflection, this doesn't quite make sense? Very happy to re-word as you see fit. I think it's worth nit-picking this, because I think we'll come back to reference it quite frequently.

Reply:
@gacafe any wording suggestions you may have would also be welcome :)

Reply:
It makes sense to me if the growth for the person is the first step of a bigger plan which has more broader impact.

I'm not necessarily saying that self-scoped projects are not good, but it might make sense, at least in the beginning, try to focus on project which have a little broader impact

Reply:
i've attempted a minor re-wording, maybe that helps @marcosh ?

@gacafe should also be able to take a look today, hopefully, or at least this week.

Reply:
I like this, nice work @silky!